apps/fluent-tester/src/testPages.ts

```diff
@@ -78,13 +78,13 @@ export const tests: TestDescription[] = [
     platforms: ['android', 'macos', 'win32'],
   },
   {
-    name: 'Checkbox',
+    name: 'Checkbox Pre-V1',
```

Comment:
I think I like "Legacy" or "v0" over "pre-v1" but naming is hard so I wouldn't call it blocking.

Reply:
Sure, I can use legacy instead.